features/admin.logs.v1/components/remote-logging-config-form.tsx

```diff
@@ -0,0 +1,411 @@
+                            "description"
+                        ),
+                        level: AlertLevels.SUCCESS,
+                        message: t(
+                            "console:manage.features.serverConfigs.remoteLogPublishing." +
+                            "notification.success.message"
+                        )
+                    })
+                );
+            })
+            .catch((_err: AxiosError) => {
+                dispatch(
+                    addAlert<AlertInterface>({
+                        description: t(
+                            "console:manage.features.serverConfigs.remoteLogPublishing." +
+                            "notification.error.updateError.description"
+                        ),
+                        level: AlertLevels.ERROR,
+                        message: t(
+                            "console:manage.features.serverConfigs.remoteLogPublishing." +
+                            "notification.error.updateError.message"
+                        )
+                    })
+                );
+            });
+    };
+
+    if (isLoading) {
+        return (
+            <Card className="remote-logging-content">
+                <div className="remote-logging-form">
+                    <Stack direction="column" spacing={ 2 }>
+                        <Skeleton variant="rectangular" height={ 7 } width="30%"/>
+                        <Skeleton variant="rectangular" height={ 28 }/>
+                        <Skeleton variant="rectangular" height={ 7 } width="90%"/>
+                        <Skeleton variant="rectangular" height={ 7 }/>
+                    </Stack>
+                </div>
+            </Card>
+        );
+    }
+
+    function handleTestNow() {
+        setTestNow(true);
+        mutateTest();
+    }
+
+    return (
+        <>
+            <Card className="remote-logging-content">
+                <div className="remote-logging-form">
+                    <Grid xs={ 12 } md={ 8 } lg={ 4 }>
+                        <Forms onSubmit={ handleRemoteLoggingConfigUpdate } resetState={ resetForm }>
+                            <Field
+                                label={ t(
+                                    "console:manage.features.serverConfigs.remoteLogPublishing.fields.remoteURL.label"
+                                ) }
+                                name={ "remoteUrl" }
+                                required
+                                requiredErrorMessage={ t(
+                                    "console:manage.features.serverConfigs.remoteLogPublishing.fields.remoteURL" +
+                                    ".remoteURL.error.required"
+                                ) }
+                                type="text"
+                                value={ logConfig?.url }
+                                data-componentid={ componentId + "-url-value-input" }
```

Comment:
Let's flow the Template literal approach. It is more readable and consistent with other places.
`${componentId}-url-value-input`